MetaMask/metamask-mobile
# feat: Move Security settings from Experimental Settings into Security Settings

## **Description**

We should move security alerts feature out of Settings > Experimental to Settings > Security & Privacy under the Security section below the show private key feature.

## **Related issues**

Fixes: [#1986](https://github.com/MetaMask/MetaMask-planning/issues/1986)

## **Manual testing steps**

1. Start Metamask
2. Open Settings > Experimental and see blockaid settings
3. Checkout this branch
4. Open Metamask
5. Open Settings > Security & Privacy and see blockaid settings under private key settings

## **Screenshots/Recordings**

### **Before**

https://github.com/MetaMask/metamask-mobile/assets/44811/0ed7131a-fcb4-4e57-b697-83496aa73d5b

### **After**

https://github.com/MetaMask/metamask-mobile/assets/44811/2e68562d-1674-41fe-9db9-3270d1bffb9b


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
